code/game/objects/items/devices/PDA/apps/general.dm

```diff
@@ -142,7 +142,7 @@ var/global/list/currentevents3 = list("Border patrol around Space America has ti
 	"Renowned mime scientist Free Shrugs has discovered a new element today. He has named it '  ', he also says that it has the properties of '   '.",
 	"Archaeologists have discovered god's final message to his creation today. The message reads, 'bawk'.",
 	"Scientists have discovered a new type of elementary particle today. Our sources say it has a bad atitude, and enjoys the color blue.",
-	"Today, a man was discovered to be living with a 20 year old ghost in his house. When the ghost was questioned who killed him, he responded 'A FAGGOT!'. More at four.",
+	"Today, a man was discovered to be living with a 20 year old ghost in his house. When the ghost was questioned who killed him, he responded 'A Tajaran!'. More at four.",
```

Comment:
```suggestion
	"Today, a man was discovered to be living with a 20 year old ghost in his house. When the ghost was questioned who killed him, he responded 'A BASTARD!'. More at four.",
```
Sorta keeps the vibe a bit more imo. What do you think? Plenty of mainstream games have bastard used as an insult.

Reply:
This is very problematic and bigoted against people with no fathers.

Reply:
get out of here ada

Reply:
this is not in the gamer compliance list